internal/storage/v1/elasticsearch/spanstore/internal/dbmodel/model.go

```diff
@@ -100,3 +102,15 @@ type OperationQueryParameters struct {
 	ServiceName string
 	SpanKind    string
 }
+
+// TraceQueryParameters contains parameters of a trace query.
+type TraceQueryParameters struct {
+	ServiceName   string
+	OperationName string
+	Tags          map[string]string
+	StartTimeMin  time.Time
+	StartTimeMax  time.Time
+	DurationMin   time.Duration
+	DurationMax   time.Duration
+	NumTraces     int
```

Comment:
Let's add a TODO to rename to `SearchDepth`

https://github.com/jaegertracing/jaeger/blob/4af5162098afa29ea588a7da3934d5856d9b8f84/internal/storage/v2/grpc/trace_storage.proto#L104

cc @Manik2708